README.md

```diff
@@ -62,8 +62,17 @@ Request approval for a new subscription by emailing "DTS Platform Operations"
 
 Once approved create a pull request adding the required subscriptions.
 
-1. Modify the file [prod.tfvars](https://github.com/hmcts/azure-enterprise/blob/main/environments/prod/prod.tfvars) with the subscription name.
+1. Modify the file [prod.tfvars](https://github.com/hmcts/azure-enterprise/blob/main/environments/prod/prod.tfvars) with the subscription name and its environment.
    * add it into the corresponding management group e.g. for a new heritage production subscription add it to the `heritage_production_subscriptions` variable.
+   ```
+   heritage_production_subscriptions = {
+    DTS-HERITAGENEWSUB-STG = {
+      environment = "stg
```

Comment:
I think for 90% of subscriptions you can detect this with a regex, and should allow specifying it to override with subscriptions not matching the naming convention